py/src/braintrust/wrappers/test_openai.py

```diff
@@ -986,18 +985,17 @@ class NumberAnswer(BaseModel):
         {
             "input": TEST_PROMPT,
             "metadata": {
-                "model": TEST_MODEL,
+                "model": lambda x: TEST_MODEL in x,
```

Comment:
would be nice if we did some assertions against the missing bits that were missing